recipes/porechop/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: porechop
+  version: '0.2.3_seqan2.1.1'
```

Comment:
I'm wondering if you want to encode this in the build-string?

Reply:
I'm not sure -- see below.